src/ontology/cl-edit.owl

```diff
@@ -8269,9 +8269,10 @@ SubClassOf(obo:CL_0000567 obo:CL_0000198)
 
 # Class: obo:CL_0000568 (amine precursor uptake and decarboxylation cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0721662544") Annotation(oboInOwl:hasDbXref "MESH:A11.070") obo:IAO_0000115 obo:CL_0000568 "A cell that originates in the neural crest, that has certain cytochemical and ultrastructural characteristics and is found scattered throughout the body; types include melanocytes, the cells of the chromaffin system, and cells in the hypothalamus, hypophysis, thyroid, parathyroids, lungs, gastrointestinal tract, and pancreas. This cell type concentrates the amino acid precursors of certain amines and decarboxylate them, forming amines that function as regulators and neurotransmitters. This cell type produces substances such as epinephrine, norepinephrine, dopamine, serotonin, enkephalin, somatostatin, neurotensin, and substance P, the actions of which may affect contiguous cells, nearby groups of cells, or distant cells, thus functioning as local or systemic hormones. The name is an acronym for amine precursor uptake and decarboxylation cell.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0721662544") obo:IAO_0000115 obo:CL_0000568 "A cell that originates in the neural crest, that has certain cytochemical and ultrastructural characteristics and is found scattered throughout the body; types include melanocytes, the cells of the chromaffin system, and cells in the hypothalamus, hypophysis, thyroid, parathyroids, lungs, gastrointestinal tract, and pancreas. This cell type concentrates the amino acid precursors of certain amines and decarboxylate them, forming amines that function as regulators and neurotransmitters. This cell type produces substances such as epinephrine, norepinephrine, dopamine, serotonin, enkephalin, somatostatin, neurotensin, and substance P, the actions of which may affect contiguous cells, nearby groups of cells, or distant cells, thus functioning as local or systemic hormones. The name is an acronym for amine precursor uptake and decarboxylation cell.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000568 "BTO:0003866")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000568 "FMA:83114")
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000568 "MESH:D001078")
```

Comment:
Is this DbXref supposed to be annotated to the defintion?

Reply:
My interpretation of the [conversation thread here](https://github.com/obophenotype/cell-ontology/issues/1557) is that there should be an effort to move these references to the class/node level.
Also, the MESH:[UniqueID] links do not direct to a page with a text definition: https://id.nlm.nih.gov/mesh/[UniqueID].html.
However, links with the format https://meshb.nlm.nih.gov/record/ui?ui=[UniqueID] (not the same as the MESH prefix) do include information for text definitions.